### PR TITLE
Abstract

### PR DESCRIPTION
**Pull request by boucadair**

### CODE REVIEW

**Review by SpencerDawkins — COMMENTED** [review]
This is a nice clean-up, but still refers to requirements. If we're going to do requirements, that's going to be a lot more work than explaining ATSSS to the IETF community so they can work on requirements without arguing with us about every requirement ... and that's a global thing, not limited to the Abstract, so we need to converge on a decision. 
Do the right thing, of course.

File: draft-quic-atsss-reqs.mkd [hunk 1]
```diff
@@ -55,7 +55,7 @@ informative:
 
 --- abstract
 
-This document briefly presents the Access Traffic Steering, Switching, and Splitting (ATSSS) service being specified within the 3rd Generation Partnership Project (3GPP). The document also captures a set of data plane requirements to benefit from multipath capabilities. 
+This document briefly presents the Access Traffic Steering, Switching, and Splitting (ATSSS) service being specified within the 3rd Generation Partnership Project (3GPP). Future verssions of the document will capture a set of data plane requirements to benefit from multipath capabilities.
```
Comment (obonaventure) [inline draft-quic-atsss-reqs.mkd]: verssions -> versions besides that fine for me

Comment (boucadair) [reply draft-quic-atsss-reqs.mkd]: Cool. The nit was fixed here: https://github.com/obonaventure/draft-quic-atsss-reqs/pull/8/commits/fe473cd2409a2fd068f41be1a89c54981019eecc

**Review by mpiraux — APPROVED** [review]
LGTM

**Pull request merged**